community/kernel/src/test/java/org/neo4j/kernel/impl/storemigration/participant/StoreMigratorIT.java

```diff
@@ -25,12 +25,15 @@
 import org.junit.runners.Parameterized;
 
 import java.io.File;
+import java.io.IOException;
```

Comment:
indeed, sorry for that and strangely intelij does not cleaned them up...
should be clean now